packages/lodestar/src/tasks/index.ts

```diff
@@ -50,14 +50,25 @@ export class TasksService implements IService {
 
   public async start(): Promise<void> {
     this.chain.on("finalizedCheckpoint", this.handleFinalizedCheckpointChores);
-    await this.interopSubnetsTask.start();
+    this.network.gossip.on("gossip:start", this.handleGossipStart);
+    this.network.gossip.on("gossip:stop", this.handleGossipStop);
   }
 
   public async stop(): Promise<void> {
     this.chain.removeListener("finalizedCheckpoint", this.handleFinalizedCheckpointChores);
+    this.network.gossip.removeListener("gossip:start", this.handleGossipStart);
+    this.network.gossip.removeListener("gossip:stop", this.handleGossipStop);
     await this.interopSubnetsTask.stop();
```

Comment:
remove this as `this.interopSubnetsTask` is removed in `handleGossipStop`

Reply:
We want the interopSubnetsTask to shut down _regardless_, when task manager is shutdown.
In practice, the tasks manager is shutting down before the network, too.